apps/website/src/pages/_app.js

```diff
@@ -46,7 +44,6 @@ function MyApp(props) {
     <>
       <SheetsProvider>
         <ThemeProvider theme={theme}>
-          <GlobalStyle />
```

Comment:
i dont know @Enjoy2Live has made this change but i guess `<GlobalStyles/>` imports styled component css configs and its not needed anymore since we're using tailwind

Reply:
Got it. So this change should not have any effect then?

Reply:
no it shouldnt

Reply:
thanks for confirming!